File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ClientGroupListModel.ts
export interface ClientGroupListModel {
    clientGroupId: number;
    clientGroupCode: string;
    clientGroupName: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    icon: string;
    color: string;
    displayOrder: number | null;
}

File: SofiForce.Models/Models/ListModels/ClientGroupListModel.cs
using System;

namespace Models
{
    public class ClientGroupListModel
    {
        public int ClientGroupId { get; set; }
        public string ClientGroupCode { get; set; }
        public string ClientGroupName { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
    }
}
